observation_portal/observations/viewsets.py

```diff
@@ -49,6 +56,82 @@ def observations_queryset(request):
     ).distinct()
 
 
+class RealTimeViewSet(CreateListModelMixin, viewsets.ModelViewSet):
```

Comment:
Oh one thing I think I forgot is that we need a way for the UI to hit an endpoint to get the set of available session slots on the network +/- 2 hours (configurable) from the time they set. So we'd need to have configuration for 

1) datetime intervals in the night (first and last 2 hours on each site?) that we allow RTI observations in 
2) a set of instrument telescope classes we allow these types of observations on

Maybe that's outside the scope of this? Or would it go better in the LCO obs portal project? We could definitely get something working now without all of this, but I think an availability endpoint that we can add more complexity to would be nice.

Reply:
Yeah I didn't do that yet since we haven't really defined what we want from it... I kind of want to flesh out what limits we want to place on it first, and then that might dictate where it should go (i.e. in the base if its configurable enough to be general, or in the lco project if its really specific stuff).